packages/runtime-dom/types/jsx.d.ts

```diff
@@ -1,1345 +1 @@
-  onBlur: FocusEvent
-
-  // form events
-  onChange: Event
-  onBeforeinput: Event
-  onInput: Event
-  onReset: Event
-  onSubmit: Event
-  onInvalid: Event
-
-  // image events
-  onLoad: Event
-  onError: Event
-
-  // keyboard events
-  onKeydown: KeyboardEvent
-  onKeypress: KeyboardEvent
-  onKeyup: KeyboardEvent
-
-  // mouse events
-  onAuxclick: MouseEvent
-  onClick: MouseEvent
-  onContextmenu: MouseEvent
-  onDblclick: MouseEvent
-  onMousedown: MouseEvent
-  onMouseenter: MouseEvent
-  onMouseleave: MouseEvent
-  onMousemove: MouseEvent
-  onMouseout: MouseEvent
-  onMouseover: MouseEvent
-  onMouseup: MouseEvent
-
-  // media events
-  onAbort: Event
-  onCanplay: Event
-  onCanplaythrough: Event
-  onDurationchange: Event
-  onEmptied: Event
-  onEncrypted: Event
-  onEnded: Event
-  onLoadeddata: Event
-  onLoadedmetadata: Event
-  onLoadstart: Event
-  onPause: Event
-  onPlay: Event
-  onPlaying: Event
-  onProgress: Event
-  onRatechange: Event
-  onSeeked: Event
-  onSeeking: Event
-  onStalled: Event
-  onSuspend: Event
-  onTimeupdate: Event
-  onVolumechange: Event
-  onWaiting: Event
-
-  // selection events
-  onSelect: Event
-
-  // UI events
-  onScroll: UIEvent
-
-  // touch events
-  onTouchcancel: TouchEvent
-  onTouchend: TouchEvent
-  onTouchmove: TouchEvent
-  onTouchstart: TouchEvent
-
-  // pointer events
-  onPointerdown: PointerEvent
-  onPointermove: PointerEvent
-  onPointerup: PointerEvent
-  onPointercancel: PointerEvent
-  onPointerenter: PointerEvent
-  onPointerleave: PointerEvent
-  onPointerover: PointerEvent
-  onPointerout: PointerEvent
-
-  // wheel events
-  onWheel: WheelEvent
-
-  // animation events
-  onAnimationstart: AnimationEvent
-  onAnimationend: AnimationEvent
-  onAnimationiteration: AnimationEvent
-
-  // transition events
-  onTransitionend: TransitionEvent
-  onTransitionstart: TransitionEvent
-}
-
-type EventHandlers<E> = {
-  [K in keyof E]?: E[K] extends (...args: any) => any ? E[K] : (payload: E[K]) => void
-}
-
-// use namespace import to avoid collision with generated types which use
-// named imports.
-import * as RuntimeCore from '@vue/runtime-core'
-
-type ReservedProps = {
-  key?: string | number | symbol
-  ref?: RuntimeCore.VNodeRef
-  ref_for?: boolean
-  ref_key?: string
-}
-
-type ElementAttrs<T> = T & ReservedProps
-
-type NativeElements = {
-  [K in keyof IntrinsicElementAttributes]: ElementAttrs<
-    IntrinsicElementAttributes[K]
-  >
-}
-
-declare global {
-  namespace JSX {
-    interface Element extends VNode {}
-    interface ElementClass {
-      $props: {}
-    }
-    interface ElementAttributesProperty {
-      $props: {}
-    }
-    interface IntrinsicElements extends NativeElements {
-      // allow arbitrary elements
-      // @ts-ignore suppress ts:2374 = Duplicate string index signature.
-      [name: string]: any
-    }
-    interface IntrinsicAttributes extends ReservedProps {}
-  }
-}
-
-// suppress ts:2669
-export {}
+import '@vue/jsx/register'
```

Comment:
This produces a warning that `./dist/jsx` does not exist. Since we are building in parallel, it is eventually created.